packages/ReadWrite/Guard/VariableToConstantGuard.php

```diff
@@ -103,7 +103,7 @@ private function resolveFunctionReferencePositions(
         return $referencePositions;
     }
 
-    private function getArgumentPosition(FuncCall $funcCall, Arg $desiredArg): int
+    private function getArgumentPosition(FuncCall $funcCall, Arg $desiredArg): int|string
```

Comment:
@TomasVotruba it seems invalid, as func call args key should be int

Reply:
I'll check it, thanks

Reply:
I think problem is PHPStan sees all keys as int|string unless defined otherwise. Here the key is not defined for $funcCall->args.

But I can add `@var int` manually here to mitigate it

Reply:
Ref https://github.com/rectorphp/rector-src/pull/1569